README.md

```diff
@@ -18,6 +18,23 @@ $ npm install -g gitlab-ci-variables-setter-cli
 
 ## Usage
 
+### One variable (`glci sv`)
+
+Run the following command, where:
+
+- `gitlab-token` is your Gitlab personal access token
+- `gitlab-project-url` is your project url on gitlab, e.g. https://gitlab.com/gitlab-org/gitlab-ce
+- `gitlab-key` is the variable you want to set
+- `gitlab-value` is the value of the variable you want to set
+
+```sh
+$ glci sv --token <gitlab-token> --url <gitlab-project-url> --key <gitlab-key> --value <gitlab-value>
```

Comment:
Below we just use `key`, `value`, so perhaps the placeholders could be updated to reflect the same.

Reply:
fixed